src/improvements/tasks/StateOverrideManager.sol

```diff
@@ -97,6 +97,16 @@ abstract contract StateOverrideManager is CommonBase {
         defaultOverride = Simulation.addOwnerOverride(parentMultisig, defaultOverride, owner);
     }
 
+    /// @notice Parent multisig override for nested execution.
```

Comment:
For nested simulations, we should not perform the owner override.

Reply:
For single simulations, we do perform the owner override on the parent multisig so we can execute the transaction from the sender.

Reply:
nit: it seems like `_ownerMultisig` might be slightly more accurate, as it's not really a parent in the single case, but I think it's probably not worth changing right now.

Reply:
Yea I think we started using parent and child in the new scops and now I keep using it.